config/gearman.php

```diff
@@ -1,35 +1,21 @@
 <?php
-// Copyright (C) <2015>  <it-novum GmbH>
-//
-// This file is dual licensed
-//
-// 1.
-//	This program is free software: you can redistribute it and/or modify
-//	it under the terms of the GNU General Public License as published by
-//	the Free Software Foundation, version 3 of the License.
-//
-//	This program is distributed in the hope that it will be useful,
-//	but WITHOUT ANY WARRANTY; without even the implied warranty of
-//	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-//	GNU General Public License for more details.
-//
-//	You should have received a copy of the GNU General Public License
-//	along with this program.  If not, see <http://www.gnu.org/licenses/>.
-//
 
-// 2.
-//	If you purchased an openITCOCKPIT Enterprise Edition you can use this file
-//	under the terms of the openITCOCKPIT Enterprise Edition license agreement.
-//	License agreement and license key will be shipped with the order
-//	confirmation.
+//////////////////////////////////////////////////////////////////////////
+//    DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN  //
+//                                                                       //
+//                   File generated by openITCOCKPIT                     //
+//                                                                       //
+//                        Created: 09.03.2021 08:01                      //
+//////////////////////////////////////////////////////////////////////////
+
 
 return [
     'gearman' => [
         'address'    => '127.0.0.1',
         'port'       => 4730,
-        'encryption' => true,
+        'encryption' => 1,
         'password'   => '3c3b0be215c95321c30de1ab908364dfbf0ed440',
         'pidfile'    => '/var/run/oitc_gearmanworker.pid',
-        'worker'     => 5,
-    ],
-];
+        'worker'     => 5
+    ]
+];
```

Comment:
Please delete this config file form the repositoriy otherwise it's a change whenever the configuration get's regenerated.